tests/units/cli/resources/test_yaml_config_with_slack_channel/config.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+alert_handlers:
+  slack:
+    token: !vault |
+      $ANSIBLE_VAULT;1.1;AES256
+      39353938316661653737376632333765636234643638613464303564393431626133313238353334
+      3666316661373931363839346630626662393664633731340a613366333563386536316363313432
+      37636531343162656430396630383736323564363637303730613133316139633964303466306364
+      6231336535353465360a636262316639336134366337373033303265383235393638383035366633
+      37643039346438666439616332626430323531363933663430633662323936633630
```

Comment:
can you please remove and use something dummy? We should also remove similar thing from https://github.com/transferwise/pipelinewise/blob/ac926d7fe0322e8382850f4889fac85cd316384f/tests/units/cli/resources/test_yaml_config/config.yml

Reply:
- [ ] Use a dummy value